go/fixchain/hash.go

```diff
@@ -25,6 +25,10 @@ func hashChain(ch []*x509.Certificate) (hash [hashSize]byte) {
 	return
 }
 
+// hashBag hashes all of the certs in the chain, irrespective of their order.
+// Two chains containing the same certs in different orders will result in the
+// same hash.  Two chains containing the same certs but of different lengths
```

Comment:
This not a technically accurate description (i.e. they could be the same length, but still differ). But I won't make you change it, I get the drift.